src/containerapp/azext_containerapp/_params.py

```diff
@@ -177,6 +177,7 @@ def load_arguments(self, _):
         c.argument('platform_reserved_cidr', options_list=['--platform-reserved-cidr'], help='IP range in CIDR notation that can be reserved for environment infrastructure IP addresses. It must not overlap with any other Subnet IP ranges')
         c.argument('platform_reserved_dns_ip', options_list=['--platform-reserved-dns-ip'], help='An IP address from the IP range defined by Platform Reserved CIDR that will be reserved for the internal DNS server.')
         c.argument('internal_only', arg_type=get_three_state_flag(), options_list=['--internal-only'], help='Boolean indicating the environment only has an internal load balancer. These environments do not have a public static IP resource, therefore must provide infrastructureSubnetResourceId if enabling this property')
+        c.argument('infrastructure_resource_group', options_list=['--infrastructure-resource-group', '-i'], help='Name for resource group that will contain infrastructure resources. If not provided, the default resource group name will be used.')
```

Comment:
Does this mean that the rg set using  `az configure --defaults group=<name>` will be used or just a randomally generated rg using our default values be used? Because i tested it and it used a randomly generated name not the default rg set by az configured? Is that expected

Reply:
This is a different resource group from the one default would set, `az configure` would set the one corresponding to [this parameter](https://learn.microsoft.com/en-us/rest/api/containerapps/preview/managed-environments/create-or-update?tabs=HTTP#:~:text=of%20the%20Environment.-,resourceGroupName,-path), whereas the new field maps onto [this parameter](https://learn.microsoft.com/en-us/rest/api/containerapps/preview/managed-environments/create-or-update?tabs=HTTP#:~:text=of%20Dapr%20component.-,properties.infrastructureResourceGroup,-string).

Reply:
both links go to the same page for me. But I get know that az configure --defaults group=<name> is not setting the same thing.

maybe then say "if not provided a resource group name will be generated?"

Reply:
I like that, added.